moon/rospy/rospy_round1.py

```diff
@@ -12,11 +14,11 @@
 
 
 class RospyRound1PushPull(RospyBasePushPull):
-        
+
     def register_handlers(self):
         super(RospyRound1PushPull, self).register_handlers()
-        
-        rospy.Subscriber('/qual_1_score', Qual1ScoringMsg, self.callback_topic, '/qual_1_score')
+
+        rospy.Subscriber('/qual_1_score', Qual1ScoringMsg, partial(self.callback_topic, rate=1000), '/qual_1_score')
```

Comment:
isn't this obsolete now? NASA planned to downsample score for the final release - in that case I would drop this functionality.

Reply:
They changed the frequency from 1kHz to 50Hz. Probably something we could live with performance-wise but I feel we can still downsample to get to 1Hz where we were before.